### PR TITLE
GH-1158 Fixes Surface button annotations for FF

### PR DESCRIPTION
**Pull request by mlzummo**
Fixes GH-1158

Signed-off-by: Jay Brundage <jaman.brundage@lexisnexis.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
dont pull yet need to fix 1 more thing

**Comment by mlzummo** [comment]
@GordonSmith im not sure what to do with the .classed here in the if statement?

File: src/layout/Surface.js [hunk 1]
```diff
@@ -91,12 +91,15 @@
                     .style("width", button.width)
                     .style("height", button.height)
                     .style("cursor","pointer");
+                if(button.class){
+                    el.classed(button.class,true);
+                }
```
Comment (GordonSmith) [inline src/layout/Surface.js]: This is probably not needed now?

**Comment by GordonSmith** [comment]
The bit I don't know is if button.class can change at runtime?

**Comment by mlzummo** [comment]
i havent a clue ... ill make the change and lets pull this and deal with that when the time comes

**Comment by mlzummo** [comment]
does this code need to be changed in the other Surface.js?

**Comment by GordonSmith** [comment]
I don't think so as its buttons are just SVG Icon Widgets

**Comment by mlzummo** [comment]
@GordonSmith status update on pulling this?

**Pull request merged**